src/index.ts

```diff
@@ -1,26 +1,42 @@
 import express, { Request, Response } from 'express';
 import db from './knex';
-//comment
+import cors from 'cors';
+import dotenv from 'dotenv';
+import jwt from 'jsonwebtoken';
+
+dotenv.config();
 const app = express();
 const port = 3000;
 
+const JWT_SECRET = process.env.JWT_SECRET || 'key';
+
 // Middleware
 app.use(express.json());
+app.use(cors({ origin: 'http://localhost:5173' }));
 
-// Basic GET Route
-app.get('/', (req: Request, res: Response) => {
-  res.send('Hello World!');
-});
 
-app.get('/users', async (req: Request, res: Response) => {
+// Creating new users -endpoint
+app.post('/users', async (req: Request, res: Response) => {
+  const { email, password } = req.body;
+
+  if (!email || !password) {
+    return res.status(400).json({ message: 'Email and password are required' });
+  }
+
   try {
-    const users = await db('table-1').select('*');
-    res.json(users);
-  } catch (error) {
-    res.status(500).json({ message: 'Error fetching users', error });
+    const [userId] = await db('users').insert({ email, password }).returning('id');
+    const token = jwt.sign({ userId }, JWT_SECRET, { expiresIn: '1h' });
+    res.status(201).json({ message: 'User created successfully', token, userId });
```

Comment:
```suggestion
    res.status(201).json({ message: 'User created successfully', token, userId });
```

We don't need to send any `message` to the client. `201` already means it is created. 
No need to send `userId`. But we should send `email` instead.